influxql/parser.go

```diff
@@ -311,8 +313,8 @@ func (p *Parser) parseCreateRetentionPolicyStatement() (*CreateRetentionPolicySt
 	// Parse optional DEFAULT token.
 	if tok, pos, lit = p.scanIgnoreWhitespace(); tok == DEFAULT {
 		stmt.Default = true
-	} else {
-		p.unscan()
+	} else if tok != EOF {
+		return nil, newParseError(tokstr(tok, lit), []string{"DEFAULT"}, pos)
```

Comment:
May need to allow `SEMICOLON` token also, in case there is more than one statement.  Is there a test for multiple statements?